typespec-tests/src/main/java/com/cadl/literalservice/models/ModelOptionalLiteralOptionalLiteral.java

```diff
@@ -5,35 +5,35 @@
 package com.cadl.literalservice.models;
 
 /**
- * Defines values for ModelOptionalLiteral.
+ * Defines values for ModelOptionalLiteralOptionalLiteral.
  */
-public enum ModelOptionalLiteral {
+public enum ModelOptionalLiteralOptionalLiteral {
```

Comment:
Hi @srnagar, we now leverage TCGC to generate the name for optional literal enum. And their logic is to concat the literal value to the enum's name. e.g. the definitiion is `optionalLiteral?: "optionalLiteral"`, so another `OptionalLiteral` is append to the name. Please let me know if this is acceptable or not.